lib/language_pack/ruby.rb

```diff
@@ -224,21 +224,8 @@ def default_java_opts
 
   def set_jvm_max_heap
     <<-EOF
-case $(ulimit -u) in
-256)   # 1X Dyno
-  JVM_MAX_HEAP=384
-  ;;
-512)   # 2X Dyno
-  JVM_MAX_HEAP=768
-  ;;
-16384) # IX Dyno
-  JVM_MAX_HEAP=2048
-  ;;
-32768) # PX Dyno
-  JVM_MAX_HEAP=5120
-  ;;
-esac
-EOF
+    JVM_MAX_HEAP=$(expr $(awk '/MemTotal/ {print $2}' /proc/meminfo) / 1024 / 4 \* 3)
```

Comment:
we should update this to match https://github.com/heroku/heroku-buildpack-jvm-common/blob/master/opt/jvmcommon.sh#L8-L19